mod-mosaic/schemas/mosaic_order_request.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Mosaic Order Request",
+  "type": "object",
+  "javaName": "MosaicOrderRequest",
+  "properties": {
+    "orderTemplateId": {
+      "description": "UUID of the order template",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "orderData": {
+      "description": "the mosaic order",
+      "type": "object",
+      "$ref": "mosaic_order.json"
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "orderTemplateId",
```

Comment:
orderTemplateId also is not required, in this case we will use default one from settings

Reply:
removed